flaky/_flaky_plugin.py

```diff
@@ -212,48 +228,37 @@ def _handle_test_success(self, test):
         :rtype:
             `bool`
         """
-        _, _, name = self._get_test_declaration_callable_and_name(test)
```

Comment:
Can't leave a line comment higher than this, but should remove nose references from the docstring.